src/core/crypto/webauthn.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { sha3_256 } from "@noble/hashes/sha3";
+import { Signature } from "./asymmetricCrypto";
+import { Deserializer, Serializer } from "../../bcs";
+import { Hex } from "../hex";
+import { HexInput } from "../../types";
+import { P256Signature } from "./p256";
+
+export class PartialAuthenticatorAssertionResponse {
+  readonly signature: P256Signature;
```

Comment:
This should be an enum of type `WebAuthnSignature`. See 
https://github.com/aptos-labs/aptos-core/pull/10755/files#diff-aa8fc08bb2653b4289de532f8454b03f0f04426c63d30f5ccfd34b185e28bb68R61-R63